site/app/config/services.yml

```diff
@@ -3,9 +3,18 @@
 parameters:
 #    parameter_name: value
 
+imports:
+    - { resource: legacy_aliases.yml }
+
```

Comment:
When removing the only content in this file above you can remove this import as well.